templates/master/00-master/_base/units/openvswitch.service

```diff
@@ -0,0 +1,2 @@
+name: "openvswitch.service"
+enabled: {{if .NetworkType == "OVNKubernetes"}}true{{else if .NetworkType == "ovs"}}true{{else}}false{{end}}
```

Comment:
who knows that the const is for OVS? is there even one? couldn't find it in openshift/api /cc @smarterclayton @mccv1r0

Reply:
Sorry, it's OpenShiftSDN https://github.com/openshift/api/blob/master/operator/v1/types_network.go#L348 and https://github.com/openshift/api/blob/master/operator/v1/types_network.go#L352